packages/next-codemod/transforms/lib/async-request-api/utils.ts

```diff
@@ -492,3 +492,66 @@ export function getVariableDeclaratorId(
   }
   return undefined
 }
+
+export function findFunctionBody(path: ASTPath<FunctionScope>): null | any[] {
+  let functionBody = path.node.body
+  if (functionBody && functionBody.type === 'BlockStatement') {
+    return functionBody.body
+  }
+  return null
+}
+
+export function containsReactHooksCallExpressions(
+  path: ASTPath<FunctionScope>,
+  j: API['jscodeshift']
+) {
+  const hasReactHooks =
+    j(path)
+      .find(j.CallExpression)
+      .filter((callPath) => {
+        return (
+          j.Identifier.check(callPath.value.callee) &&
+          callPath.value.callee.name.startsWith('use')
+        )
```

Comment:
Missing support for  `MemberExpression` e.g. `React.use()`. In React, we check if the property is a hook and the object is in PascalCase to avoid false-positives for e.g. `jest.useFakeTimers`

Reply:
Ignored the note about PascalCase. I didn't say "just `React`".